docs/content/deployment/run-monitoring.mdx

```diff
@@ -34,6 +54,9 @@ It's possible for a run worker process to crash during a run. This can happen fo
 
 If a run worker crashes, the run it's managing can hang. The monitoring daemon can run health checks on run workers for all active runs to detect this. If a failed run worker is detected (e.g. by the K8s Job having a non-zero exit code), the run is either marked as failed or resumed (see below).
 
+Note that detecting run worker crashes only works when using a run launcher other than the <PyObject object="DefaultRunLauncher" />.
```

Comment:
Thoughts on putting this into a callout right after this section's heading? I think it'd be good to tell the user this info before they read the entire section, so they can choose to skip it if they know it doesn't apply to them.

Ex:

```
<Note>
Detecting run worker crashes only works when using a run launcher other than the <PyObject object="DefaultRunLauncher" />.
</Note>

Reply:
good point. Adding that change for context.